changelog/changelog2024.md

```diff
@@ -0,0 +1,4 @@
+# Changelog 2024
+
+## 4th March 2024
+* Added `PaymentCardPaymentId` field to [Outlet item](../operations/outletitems.md#outlet-item).
```

Comment:
```suggestion
## 23th May 2024
* Extended [Outlet item](../operations/outletitems.md#outlet-item) response object with `PaymentCardPaymentId`, this affects the following operations:
  * [Get outlet items](../operations/outletitems.md#get-all-outlet-items)
```
This would be more aligned with how we write it lately

@MikeAdamsMews now you are testing me right? We don't want a new changelog2024.md it should be in changelog.md